src/components/ui/search-selector/index.js

```diff
@@ -162,6 +162,8 @@ class SearchSelector extends Component {
       [`${styles.resultsLoading}`]: loading
     });
 
+    const OptionControl = optionControlComponent;
```

Comment:
@andrewcroome Could perhaps move this and the similar `const Option` definition nearer the top of this bock (under where the variables are pulled out of props/state) so they’re not so orphaned in the middle of blocks, was fine before when there was only one, but I think now there are more it makes sense to have them defined together so you can easily see them.

Reply:
Thanks @makenosound. I shall do.